pkg/apiserver/apiserver_test.go

```diff
@@ -144,6 +152,11 @@ func addTestTypes() {
 	api.Scheme.AddKnownTypes(testInternalGroupVersion,
 		&apiservertesting.Simple{}, &apiservertesting.SimpleList{}, &api.ListOptions{},
 		&apiservertesting.SimpleGetOptions{}, &apiservertesting.SimpleRoot{})
+	api.Scheme.AddKnownTypes(testGroup2Version,
+		&SimpleXGSubresource{})
+	api.Scheme.AddKnownTypes(testGroupVersion, &api.Pod{})
+	api.Scheme.AddKnownTypes(testInternalGroup2Version,
+		&SimpleXGSubresource{})
```

Comment:
Maybe register this in the other group, too, to prove that the correct one is chosen.